sklearn/metrics/tests/test_classification.py

```diff
@@ -160,7 +144,6 @@ def test_precision_recall_f1_score_binary():
                             (1 + 2 ** 2) * ps * rs / (2 ** 2 * ps + rs), 2)
 
 
-@ignore_warnings
```

Comment:
Why? Are there no longer any warnings from this test? I don't think it has anything to do with seq-of-seqs multilabel, given the name "`binary_single_class`"... but `ignore_warnings` is also used for other things below.